src/Schema/Table.php

```diff
@@ -357,6 +359,26 @@ public function addColumn($name, $typeName, array $options = [])
      * @throws SchemaException
      */
     public function changeColumn($name, array $options)
+    {
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/pull/5747',
+            '%s is deprecated. Use modifyColumn() instead.',
+            __METHOD__,
+        );
+
+        return $this->modifyColumn($name, $options);
+    }
+
+    /**
+     * @param string  $name
+     * @param mixed[] $options
+     *
+     * @return self
+     *
+     * @throws SchemaException
+     */
+    public function modifyColumn($name, array $options)
```

Comment:
Because it's too hard to think of where we can and where we cannot enforce types at this point. In the absence of automation in this regard in `3.5.x` the mental effort required to make it perfect is not worth the result. It will all be correct in `4.0.x` (I guess my answer was along the same lines last time).